docs/source/guide/install.md

```diff
@@ -84,7 +84,7 @@ label-studio
 The default web browser will automatically open at [http://localhost:8080](http://localhost:8080) with Label Studio. Please refer to [start Label Studio](start.html) for more options when starting Label Studio.
 
 !!! note
-    If you've previously installed the brew tap from the now-deprecated organization name `heartexlabs/tap`, we've got good news. You don't have to worry about migrating immediately. The deprecated tap has been set up as a mirror for `humansignal/tap`. This ensures continuity and minimizes disruptions for existing users.
+If you've previously installed the brew tap from the now-deprecated organization name `heartexlabs/tap`, we've got good news. You don't have to worry about migrating immediately. The deprecated tap has been set up as a mirror for `humansignal/tap`. This ensures continuity and minimizes disruptions for existing users.
 
```

Comment:
@huguestennier - Did you mean to remove the spaces? Doesn't that break the note output? https://deploy-preview-4744--label-studio-docs-new-theme.netlify.app/guide/install#Install-using-Homebrew

Reply:
🤦‍♂️ Indeed! I have an auto-formatter whenever I save a file and it did that automatically. Fixing now